pubsub/aws/snssqs/snssqs.go

```diff
@@ -54,15 +55,19 @@ type snsSqsMetadata struct {
 	messageVisibilityTimeout int64
 	// number of times to resend a message after processing of that message fails before removing that message from the queue. Default: 10
 	messageRetryLimit int64
+	// if sqsDeadLettersQueueName is set to a value, then the deadLettersMaxReceives defines the number of times a message is received
+	// before it is moved to the dead-letters queue. This value must be smaller than messageRetryLimit
+	deadLettersMaxReceives int64
```

Comment:
Can this be renamed to messageReceiveLimit?

Reply:
sure. I borrowed it from the AWS official param. but your suggestion makes sense